amti/utils/workers.py

```diff
@@ -0,0 +1,40 @@
+""" Module for worker management functions """
+import boto3
+import click
+import csv
+from typing import List, Optional
+
+def create_batches(items: List, n=100) -> List:
+    """ Create generator that splits items into batches of size n. """
```

Comment:
Please remove the leading and trailing space from the first line of these doc strings, and add `Parameters` and `Returns` sections (see [this function](https://github.com/allenai/amti/blob/master/amti/actions/create.py#L19) for example).